tests/pages/products/vpn/resource_center.py

```diff
@@ -0,0 +1,26 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+from selenium.webdriver.common.by import By
+
+from pages.base import BasePage
+
+
+class VPNResourceCenterHomePage(BasePage):
+
+    _URL_TEMPLATE = "/{locale}/products/vpn/resource-center/"
+
+    # Header unit
+    _resource_center_header_locator = (By.CSS_SELECTOR, ".mzp-c-call-out.resource-center-page-header.resource-center-hero")
+
+    # Article link
+    _resource_center_article_link_locator = (By.CSS_SELECTOR, ".mzp-c-card a.mzp-c-card-block-link")
```

Comment:
Nit: this selector feels a little too non-specific. We could add a card in the navigation, or somewhere else on the page for example, and the test would pick that instead and still pass, even if no resource articles were displaying.

It would mean adding a new CSS selector to the [template's HTML](https://github.com/mozilla/bedrock/blob/main/bedrock/products/templates/products/vpn/resource-center/landing.html#L46), but something like `.resource-center-articles .mzp-c-card a.mzp-c-card-block-link` feels like a more specific target?